server/src/repositories/asset.repository.ts

```diff
@@ -525,7 +520,7 @@ export class AssetRepository {
           .$if(!!options.tagId, (qb) => withTagId(qb, options.tagId!)),
       )
       .selectFrom('assets')
-      .select(sql<string>`"timeBucket"::date::text`.as('timeBucket'))
+      .select(sql<string>`("timeBucket" AT TIME ZONE 'UTC')::date::text`.as('timeBucket'))
```

Comment:
the date needs to be represented in UTC time before being converted to a string. without this, it will use the server's timezone, which may not be UTC+0.